src/integrant/core.cljc

```diff
@@ -193,6 +203,20 @@
        (map ref-key)
        (remove #(find-derived config %))))
 
+(defn- invalid-composite-keys [config]
```

Comment:
This has two argument lists; you can remove the latter one. Also remove the docstring, since it's a private function.

Reply:
What's wrong w/ docstrings for private functions?